src/docs/overview.dox

```diff
@@ -18,8 +18,28 @@ volumes of data,
 - WiredTiger stores are not corrupted by torn writes, reverting to the
 last checkpoint after system failure,
 
-- WiredTiger supports petabyte tables, records up to 4GB, and record
-numbers up to 64-bits.
+- WiredTiger supports both checkpoint-level and commit-level durability, as well
+as an additional timestamp mode which gives applications fine-grained control
+over database durability,
+
+- WiredTiger supports petabyte tables, records up to 4GB, and record numbers up
+to 64-bits.
+
+@section quality Production quality
+
+WiredTiger is production quality, supported software, engineered for the most
+demanding application environments. WiredTiger is the primary storage engine
+under the MongoDB Database and the MongoDB  Atlas product line.
+
+As a no-overwrite data engine, torn writes can never corrupt a WiredTiger data
```

Comment:
This sentence was already there, but it's probably worth fixing. Torn writes aren't a no-overwrite data engine :-)

Perhaps "As a no-overwrite data engine, WiredTiger is safe from torn writes."

(And since I assume this means OS-level and hardware-level torn writes, might want to say that explicitly; otherwise someone is bound to think it means application writes into the database and miss the point.)

Reply:
Fixed in 5d4d2a380.